src/Data/Textile/KnittingProcess.elm

```diff
@@ -0,0 +1,91 @@
+module Data.Textile.KnittingProcess exposing
+    ( KnittingProcess(..)
+    , decode
+    , encode
+    , fromString
+    , toLabel
+    , toString
+    )
+
+import Json.Decode as Decode exposing (Decoder)
+import Json.Decode.Extra as DE
+import Json.Encode as Encode
+
+
+type KnittingProcess
```

Comment:
This could be just `Knitting` or maybe `Kind` (but I like the main type being named the same as the module)